### PR TITLE
Profiles fixes

### LINKED ISSUE #413: Profiles moniker flag take full-node moniker as default value
## Bug description
If we try to save a profile with the `desmos tx profiles save` transaction, without specifying the `--moniker` flag, the module will not use its default `[do-not-modify]` one. Instead, he will use the moniker of the full node.
## Steps to reproduce
`desmos tx profiles save <dtag>...`
check the tx info before sending it.

## Expected behavior
It should take the default flag and not the full node moniker.

### LINKED ISSUE #414: Saving a profile the first time stores `[do-not-modify]` info
## Bug description
If you use the `desmos tx profiles save Test` command the first time, the stored profile will contain `[do-not-modify]` values. 

Example can be found on tx with has `EEABCCC94F64863EBE523B9AB885E07D699FB7787F4316B048BBAE2C848B7B20` on Apollo.

## Steps to reproduce
Run `desmos tx profiles save <DTag>` from an account that does not have a profile yet. 

## Expected behavior
No `[do-not-modify]` value should exist. Empty strings should be used instead.

### PR DESCRIPTION
**Pull request by bragaz**
## Description
This PR fixes some bugs of the `x/profiles` module.
Closes #413.
Closes #414. 
## Checklist
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [ ] Wrote integration tests (simulation & CLI). 
- [ ] Updated the documentation. 
- [ ] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Review by RiccardoM — COMMENTED** [review]
Everything looks great. Could you please add a function that converts an old `Profile` into a new one? We will need this because this changes will need to be applied into the currently running testnet, and we need such function to be run during the upgrade. 

The best approach would be to follow Cosmos one as done here: 

https://github.com/cosmos/cosmos-sdk/blob/master/x/auth/keeper/migrations.go

**Comment by bragaz** [comment]
@RiccardoM I've added a legacy test that should fit the purpose, it looks like you've said that Protobuf doesn't care about fields names.

File: CHANGELOG.md [hunk 1]
```diff
@@ -110,7 +111,9 @@
 - Implemented users `Relationships` ([\#168](https://github.com/desmos-labs/desmos/issues/168))
 
 # Version 0.10.0
+
```
Comment (RiccardoM) [inline CHANGELOG.md:114]: @bragaz Can you please remove these new blank lines that you added? 

```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -110,7 +111,9 @@
...
 ## Changes
+
```
Comment (RiccardoM) [inline CHANGELOG.md:116]: Same here

```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -226,6 +229,7 @@
 - Removed the `gov` and `upgrade` modules as they are currently not used ([\#142](https://github.com/desmos-labs/desmos/issues/142))
 
 ## Bug fixes
+
```
Comment (RiccardoM) [inline CHANGELOG.md:232]: Same here

```suggestion
```

**Review by RiccardoM — APPROVED** [review]

**Pull request merged**